easybuild/easyconfigs/s/Subversion/Subversion-1.8.14-foss-2015a.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'Subversion'
+version = '1.8.14'
+
+homepage = 'http://subversion.apache.org/'
+description = " Subversion is an open source version control system."
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+source_urls = [
+    'http://apache.belnet.be/%(namelower)s',
+    'http://www.eu.apache.org/dist/%(namelower)s',
+    'http://www.us.apache.org/dist/%(namelower)s',
+]
+sources = [SOURCELOWER_TAR_BZ2]
+
+dependencies = [
+    ('APR', '1.5.2'),
+    ('APR-util', '1.5.4'),
+    ('SQLite', '3.8.8.1'),
+    ('zlib', '1.2.8'),
+    ('Serf', '1.3.8'),
```

Comment:
neon is replaced by Serf in svn 1.8 (neon is supported up to 1.6, Serf from 1.8) svn 1.7 is nuked.